### PR TITLE
Adds docs to classes/functions as needed

### PR DESCRIPTION
**Pull request by coyotesqrl**

### CODE REVIEW

**Comment by djblue** [comment]
Do we need to add a license header to each file?

**Comment by coyotesqrl** [comment]
> Do we need to add a license header to each file?

Are you asking if we should remove the headers because the repo has a license or if we should add the header to everything?

My thinking is that the things someone might actually want to take and repurpose, where there is any question of whether that would be a copyright violation without a license indicating how it can be reused (in our case, super-permissively), should have a license header. So we _could_ add headers to our pom and our Travis yaml file and our test inputs but no one would ever care to copy those. It's really just the source files that have anything worth copying where a question of rights might arise without us being explicit.

But I don't care that much. I could just as easily drop a license header on basically every file. 🤷‍♂

File: src/main/clojure/com/connexta/pom_lint.clj [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+; The above copyright notice and this permission notice shall be included in all
+; copies or substantial portions of the Software.
+;
+; THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+; IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+; FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+; AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+; LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+; OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+; SOFTWARE.
```
Comment (rzwiefel) [inline src/main/clojure/com/connexta/pom_lint.clj]: oh my <imaginative, but appropriate, phrase here>.

if we could leave this out of all files that would be 🆗

Comment (rzwiefel) [reply src/main/clojure/com/connexta/pom_lint.clj]: This feels like giving someone a car but absolutely destroying first gear and stripping all the teeth from it. 

If we could remove it entirely that'd be awesome. If it's completely unavoidable can we  just have a single line saying `See root LICENSE for license`

Comment (coyotesqrl) [reply src/main/clojure/com/connexta/pom_lint.clj]: Doing a little investigation...

**Comment by coyotesqrl** [comment]
License headers removed (license for repo stays, of course). Now, just a doc ticket.

**Review by djblue — APPROVED** [review]

**Review by rzwiefel — APPROVED** [review]

**Pull request merged**